File: Beginner/Valid Triangles (FLOW013)/valid.c
#include<stdio.h>

int main() { 

    int T, A, B, C;
    scanf("%d", &T);

    for(int i=1;i<=T;i++) {
        scanf("%d %d %d", &A, &B, &C);
        
        if(A+B+C==180) {
            printf("YES\n");
        }
        else {
            printf("NO\n");
        }
    }

    return 0;
    }

File: Beginner/Valid Triangles (FLOW013)/valid.java
import java.util.*;


class Codechef
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner scan=new Scanner(System.in);
		if(scan.hasNext()){
		    int T = scan.nextInt();
		    for(int i=0;i<T;i++){
		        int a = scan.nextInt();
		        int b = scan.nextInt();
		        int c = scan.nextInt();
		        if(a!=0 && b!=0 && c!=0 && (a+b+c)==180){
		            System.out.println("YES");
		        }
		        else{
		            System.out.println("NO");
		        }
		    }
        }
        scan.close();
    }
}
